.github/workflows/issue-autorespond-and-close.yml

```
@@ -0,0 +1,34 @@
+name: Respond and Close Issue
+
+on:
+  workflow_call:
+    inputs:
+      issue_number:
+        required: true
+        type: number
+      repository:
+        required: true
+        type: string
+      user_login:
+          description: Github Login for User who opened the issue.
```

```suggestion
        description: Github Login for User who opened the issue.
```
Looks like this had different indentation than the rest of the yaml which can cause problems